packages/driver/src/cypress/cy.coffee

```diff
@@ -56,6 +56,19 @@ create = (specWindow, Cypress, Cookies, state, config, log) ->
   stopped = false
   commandFns = {}
 
+  top.onerror = ->
```

Comment:
assign this and reuse it on the `get`, as opposed to pulling it off of `getOwnPropertyDescriptor` again